### DISCUSSION TITLE
How can we make remember me check box to be tick intead of fill ?

### DISCUSSION THREAD

**Discussion by zeeshantariq08** [filamentphp/filament#12714] [Help]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

i want to add check box to be tick instead of color fill in remember me , 
@leandrocfe  do you know ?

**Comment by leandrocfe** [comment]
Sorry, what does it mean?

**Reply by zeeshantariq08** [reply]
we have a remember me checkbox on login screen , check user check it is now filling color instead of tick mark 
so I want , when a user check remember me , it should be show tick mark instead of filling color in checkbox

**Reply by leandrocfe** [reply]
Maybe creating a custom theme and customizing the css via `.fi-checkbox-input` ?
https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme
